tests/x11/multi_users_dm.pm

```diff
@@ -38,11 +38,17 @@ sub ensure_graphical_target {
 }
 
 sub restart_x11 {
-    ensure_multi_user_target;
-    ensure_graphical_target;
+    my ($self, $setnologin) = @_;
+    record_soft_failure('Workaround for graphical errors on cirrus when restarting X11 -> performing reboot (see bsc#1063060)');
+    type_string "reboot\n";
+    my $timeout = get_var('UEFI') ? 140 : 80;
```

Comment:
Why should restart_x11 define a different timeout for UEFI? If UEFI should really differ although I do not know why I suggest to put it into `wait_boot` itself. At best same for `handle_login`